src/main/java/de/samply/share/client/model/centralsearch/UploadStats.java

```diff
@@ -1,15 +1,16 @@
 package de.samply.share.client.model.centralsearch;
 
-import org.simpleframework.xml.Element;
-import org.simpleframework.xml.Root;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
 
 /**
  * Represents the upload stats as received from central search.
  */
-@Root(name = "UploadStats")
+
+@JacksonXmlRootElement(localName = "UploadStats")
 public class UploadStats {
 
-  @Element(name = "LastUploadTimestamp")
+  @JacksonXmlProperty(localName = "LastUploadTimestamp")
```

Comment:
According to the documentation @JsonProperty should be used instead.

Reply:
The central search sends the information in XML format, not in Json:

https://code.mitro.dkfz.de/projects/CS/repos/centralsearch/browse/grails-app/controllers/de/dkfz/mdsearch/UploadController.groovy#218

Reply:
The property name has nothing to do with the serialisation into JSON or XML. It's just the case that Jackson is primarily a JSON lib that is also to serialise XML from the same classes. So please use the current recommended annotation and not the old one.